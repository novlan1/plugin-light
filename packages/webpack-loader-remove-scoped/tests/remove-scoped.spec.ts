import removeScoped from '../src/loader';

describe('removeScoped', () => {
  it('should work correctly', () => {
    const source = '<style lang="scss" scoped src="./css/index.scss"></style>';

    expect(removeScoped(source)).toMatchSnapshot();
  });
});
