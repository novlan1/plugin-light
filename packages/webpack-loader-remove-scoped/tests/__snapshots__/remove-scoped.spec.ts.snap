// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`removeScoped should work correctly 1`] = `"<style lang="scss"  src="./css/index.scss"></style>"`;
